Comparing source compatibility of /Users/bluefire/git/uima-uimaj-release/target/checkout/uimaj-core/target/uimaj-core-3.5.0.jar against /Users/bluefire/.m2/repository-ukp/org/apache/uima/uimaj-core/3.4.1/uimaj-core-3.4.1.jar
WARNING: You have ignored certain classes, i.e. superclasses and interfaces that could not be found on the classpath are ignored. Hence changes caused by these superclasses and interfaces are not reflected in the output.
***  MODIFIED CLASS: PUBLIC org.apache.uima.jcas.cas.AnnotationBase  (not serializable)
	***! CLASS FILE FORMAT VERSION: 61.0 <- 52.0
	+++  NEW CONSTRUCTOR: PROTECTED(+) AnnotationBase(org.apache.uima.jcas.JCas, int)
***  MODIFIED CLASS: PUBLIC org.apache.uima.jcas.tcas.Annotation  (not serializable)
	***! CLASS FILE FORMAT VERSION: 61.0 <- 52.0
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.uima.jcas.tcas.Annotation _createMarkerAnnotation(org.apache.uima.jcas.JCas, int, int)
***  MODIFIED CLASS: PUBLIC org.apache.uima.util.XMLInputSource  (not serializable)
	***! CLASS FILE FORMAT VERSION: 61.0 <- 52.0
	+++  NEW CONSTRUCTOR: PUBLIC(+) XMLInputSource(java.io.InputStream)
